src/components/ThreadEnvelope.vue

```diff
@@ -206,6 +210,16 @@ export default {
 		tags() {
 			return this.$store.getters.getEnvelopeTags(this.envelope.databaseId).filter((tag) => tag.imapLabel !== '$label1')
 		},
+		hasChangedSubject() {
+			return this.cleanSubject !== this.threadSubject
+		},
+		cleanSubject() {
+			let cleanSubject = this.envelope.subject
+			while (cleanSubject.startsWith('Re')) {
```

Comment:
you can use a regex (possibly the one from below) with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test and test for all possible prefixes